docs/user-guide/work-packages/baseline-comparison/README.md

```diff
@@ -0,0 +1,121 @@
+
+### Preset periods
+
+Baseline offers these preset time ranges:
+
+- _Yesterday_: Compare work packages to the previous day.
+
+- _Last working day_: Compare work packages to the most recent working day.
+
+- _Last week_: Compare work packages to seven working days ago.
+
+- _Last month_: Compare work packages to thirty working days ago.
+
+By default, Baseline will compare to 8 AM local time of the relevant day. You can change this to any other time of your choosing.
+
+>**Note:** These are relative comparison points, which means that _Yesterday_ will always refer to the day before the current day, and not a specific date. You can use these to set up "running" baselines that show you all changes within the past day or week.
+
+### A specific date
+
+![You can compare the present state to a specific date in the past](13-0_Baseline_specificDate.png)
+
+If you want to compare between now and a specific date in the past, you can select "a specific date" in the dropdown and select a particular date. With this option, the comparison will always be between the current state and that specific date in the past.
+
+>**Note:** You can use this to "freeze" the baseline comparison point so that the view always shows changes in comparison to that specific date, regardless of when you access it.
+
+### A custom date range
+
+![You can see changes between two dates](13-0_Baseline_dateRange.png)
+
+OpenProject also allows you to compare between two specific dates in the past. To select a custom date range, choose "between two specific dates" in the dropdown and select two dates in the date picker below.
+
+>**Note**: This is will create a fixed baseline view that will remain the same regardless of when you accesses it, since both points are fixed in the past.
+
+## Understanding the comparison results
+
+After selecting the comparison point, OpenProject presents the comparison results in table view using three icons to indicate the type of change for each work package.
+
+![Baseline enabled showing changes to the work package table](13-0_Baseline_table.png)
+
+When Baseline is enabled, you will see a legend at the top of the page which shows:
+
+![A legend is visible on top of the table when Baseline is enabled](13-0_Baseline_legend.png)
+
+- The comparison point or comparison period
+- The number of work packages added to view
+- The number of work packages removed from view
+- The number of work packages were modified
+
+
+### Change icons
+
+#### Now meets filter criteria
+
+![Icon](13-0_Baseline_nowMeets.png)
+
+Work packages that meet the filter criteria now but did not in the past are marked with an "Added" icon. These work packages were added to the current query after the selected comparison point, either because they were newly created since then or certain attributes changed such that they meet the filter criteria. 
+
+> **Note**: These do not necessarily represent _newly created_ work packages; simply those that are new to this particular view.
```

Comment:
Maybe I would add this to the last sentence "simply those that are new to this particular view because now they meet the filter criteria."